src/configuration/services/influxdb.md

```diff
@@ -35,8 +35,8 @@ if (getenv('PLATFORM_RELATIONSHIPS')) {
 	$relationships = json_decode(base64_decode($relationships), TRUE);
 
 	// For a relationship named 'timedb' referring to one endpoint.
-	if (!empty($relationships['timedb'])) {
-		foreach ($relationships['timedb'] as $endpoint) {
+	if (!empty($relationships['database'])) {
+		foreach ($relationships['database'] as $endpoint) {
```

Comment:
I can't imagine InfluxDB being the primary database on a project.  This seems like a bad use of the naming convention.